job-service/docs/pages/en-us/Getting-Started-batchworker.md

```diff
@@ -91,81 +91,184 @@ It is the responsibility of the Batch Worker plugin to repeatedly call the Batch
+
+##### NetBeans IDE
+
+NetBeans offers developers the ability to generate projects from archetypes via its GUI.
+
+###### Generate the New Batch Worker Aggregator
+
+Generate the new Batch Worker Aggregator from the `worker-batch-archetype` by following these instructions:
+
+- 'File > New Project...'
+- From the Categories Pane select Maven > From the Select Pane select 'Project from Archetype' > Click 'Next >'
+- Specify the details of the 'worker-batch-archetype'
+	- GroupId : com.hpe.caf.worker.batch
+	- ArtifactId : worker-batch-archetype
+	- Version : *WORKER-BATCH-ARCHETYPE-VERSION*
+	- Repository : http://rh7-artifactory.svs.hpeswlab.net:8081/artifactory/hpe-mirror
+	- Click 'Next >'
+- Enter Project Name, Project Location, GroupId, Version and Package of your Worker project, e.g:
+	- Project Name : worker-assetid
+	- Project Location : C:\MyWorkerProjects
+	- GroupId : com.hpe.caf.worker.batch.plugins
+	- Version : 1.0.0-SNAPSHOT
+	- Package : com.hpe.caf.worker.batch.plugins.assetid
+- Set workerName property > Click 'Finish', e.g:
+	- workerName : AssetIdWorker
+
+The foundations for your new Batch Worker is now set up. The generated project will contain the following submodules:
+
+- `<artifactId>-batch-plugin` - submodule containing the Worker's backend plugin code.
+- `<artifactId>-container` - submodule containing the Worker's container and testing configurations.
+
+### Further Information
+
+Information on worker-batch and its modules that the archetype utilizes can be found [here](https://github.hpe.com/caf/worker-batch/blob/develop/README.md).
+
+At the time this guide was written with:
+
+- Apache Maven 3.3.3
+- NetBeans 8.1
+- IntelliJ 15
+
+## Testing the Batch Worker
+
+For the purposes of getting started, this section assumes a new example Batch Worker Plugin has been created using the archetype. It assumes a batch definition comprising a list of document storage references is to be used. The `processBatch()` method of the plugin is expected to recursively split the list of document storage references (passed via the `batchDefinition`) until individual worker items for a single document storage reference can be sent to a target worker. The target worker in this case is expected to be a Document Worker.
+
+The Job Service can be employed to test the Batch Worker plugin created. The Job Service can send and monitor the progress of a batch of work sent to a Batch Worker. The message to be sent to the Batch Worker should include a batch definition comprising a list of document storage references as well as the target batch processor plugin (this is the name of the class that implements the Batch Worker plugin interface). The Job Tracking Worker will also be deployed alongside the Job Service and the logs belonging to this worker can be utilised to verify the Batch Worker plugin implementation.
+
+### Job Service and Batch Worker Deployment
+
```

Comment:
I haven't Connor, the guide talks generically about how you can use the job service to monitor batch jobs through "a down stream worker". I have spoken with Dermot and Andy, they have told me that the language-detection-worker (document-worker variant) and document-worker projects will be open sourced in the future. So I will be using the language-detection-worker as the down stream worker example and that way I will be able to validate these testing instructions.